driver/src/models/mod.rs

```diff
@@ -20,6 +20,10 @@ pub trait RollingHashable {
     fn rolling_hash(&self, nonce: i32) -> H256;
 }
 
+pub trait ConcatenatingHashable {
```

Comment:
nit: the "ing" in this term seems awkward. I think it would be more appropriate to use present tense "ConcatenateHashable"

Reply:
The goal of the term is to describe how the items are hashed together. In the rolling hashable trait this happens as `H(...H(H(i1), i2)...i_n)` whereas with concatenation it happens as `H(i1, ..., i_n)`

No strong feeling about removing the `ing` but then we should also adjust Rolling hashable. Might fit better to `RootHashable` which doesn't have an ing either.